### PR TITLE
Update myemail models [PD-1655]

### PR DESCRIPTION
**Pull request by darrint**
This is a literal implementation of the request in [PD-1655]

Tests pass except for the admin tests in myemails.

I need some direction as to where to take this next.

### CODE REVIEW

File: myemails/models.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
 class EmailSection(models.Model):
     SECTION_TYPES = (
         (1, 'Header'),
-        (2, 'Body'),
```
Comment (tdphillips) [inline myemails/models.py]: If anyone's concerned, the current event structure is completely unused. We have no risk of invalidating existing rows.

**Comment by aspidites** [comment]
Looks like according to @tdphillips' most recent comments, the ValueEvent and CronEvent models should disappear.

**Comment by tdphillips** [comment]
I'll leave that up to whoever works on it. My understanding was that none of the current models fit with the intended direction.

**Comment by darrint** [comment]
@tdphillips @aspidites I held off on removing those types until I understood better. Two questions now:

1. If there's no member user expectations to preserve, should we remove the existing admin entirely? i.e. `myemails/admin.py`

2. Is there any data migration to do at all? If not, I see what @tdphillips meant by drop existing tables.

**Comment by aspidites** [comment]
There isn't any data to migrate, unless you are expecting to prepopulate some tables. Either way, dropping tables should be fine.

File: myemails/models.py [hunk 2]
```diff
@@ -155,65 +150,22 @@ def send_email(self, for_object):
...
-
-class CreatedEvent(Event):
-    model = models.ForeignKey(ContentType,
-                              limit_choices_to={'model__in': [
-                                  'invoice'
-                              ]})
+        if len(templates) == 0:
+            return None
+        else:
+            return templates[0]
```
Comment (galitskyd) [inline myemails/models.py]: templates.first()?

Comment (tdphillips) [inline myemails/models.py]: .first() would handily duplicate the entire if/else as it returns None if there's nothing there.

Comment (galitskyd) [inline myemails/models.py]: :+1:

**Pull request merged**